airflow/operators/dagrun_operator.py

```diff
@@ -55,6 +57,14 @@ class TriggerDagRunOperator(BaseOperator):
         When reset_dag_run=False and dag run exists, DagRunAlreadyExists will be raised.
         When reset_dag_run=True and dag run exists, existing dag run will be cleared to rerun.
     :type reset_dag_run: bool
+    :param wait_for_completion: Whether or not wait for dag run completion.
```

Comment:
Would be good to tell users what's the default value here

Reply:
will add it